server/svix-server/tests/utils/common_calls.rs

```diff
@@ -208,3 +214,106 @@ pub async fn common_test_list<
 
     Ok(())
 }
+
+pub async fn try_message_attempts(
+    endpoint: &str,
+    msg_status: MessageStatus,
+    status_code: i16,
+    attempt_count: Option<usize>,
+) {
+    let mut cfg = get_default_test_config();
+    cfg.retry_schedule = (0..2)
+        .into_iter()
+        .map(|_| Duration::from_millis(1))
+        .collect();
+
+    let (client, _jh) = start_svix_server_with_cfg(&cfg);
+
+    let app_id = create_test_app(&client, "app1").await.unwrap().id;
+
+    let endp_id = create_test_endpoint(&client, &app_id, endpoint)
+        .await
+        .unwrap()
+        .id;
+
+    let _ = try_message_attempts_with_params(
+        &app_id,
+        &endp_id,
+        &client,
+        &cfg.retry_schedule,
+        msg_status,
+        status_code,
+        attempt_count,
+    );
+}
+
+pub async fn try_message_attempts_with_params(
```

Comment:
Nitpicky, but I'd probably add doc comments here -- especially since the `try_message_attempts` and `try_message_attempts_with_params` make assertions.

Reply:
At the cost of a bit more repetition, I removed these functions entirely.